training_pipeline/pipeline/schema.pbtxt

```diff
@@ -0,0 +1,64 @@
+feature {
```

Comment:
Maybe add a comment redirecting the readers to guide that shows you how to craft this file?

Reply:
sure thing